src/main/java/net/sf/jabref/gui/groups/GroupSelector.java

```diff
@@ -702,6 +712,11 @@ public void componentOpening() {
     }
 
     @Override
+    public String getSidePaneName() {
```

Comment:
Hm why do you set ``NAME`` as ``public static final`` and add this get method? Isn't this unnecessary then?